pkg/profiler/cpu/cpu.go

```diff
@@ -304,6 +308,13 @@ func (p *CPU) addUnwindTableForProcess(pid int) {
 			level.Debug(p.logger).Log("msg", "failed to add unwind table due to a procfs race", "pid", pid, "err", err)
 		} else if errors.Is(err, errTooManyExecutableMappings) {
 			level.Warn(p.logger).Log("msg", "failed to add unwind table due to having too many executable mappings", "pid", pid, "err", err)
+		} else if errors.Is(err, ErrTextSectionNotFound) {
+			if p.processErrorTracker[err]%50 == 0 {
```

Comment:
let's make this 50 at least a configurable field on the `CPU` struct, so we can potentially make it a flag if we wanted to one day